packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -254,17 +265,28 @@ export class DockerImageAsset extends CoreConstruct implements IAsset {
 
     let exclude: string[] = props.exclude || [];
 
-    const ignore = path.join(dir, '.dockerignore');
+    let ignore: string | undefined = undefined;
+    let ignorePaths: string[] = [path.join(dir, '.dockerignore')];
+    if (props.buildkitIgnore === true) {
+      ignorePaths.unshift(path.join(dir, this.dockerfilePath + '.dockerignore'));
+    }
```

Comment:
i don't have a use case for this, but it's just a change in behavior.  i think currently, if there is just a `file.dockerignore` file, then `file.dockerignore` will get included in the image if it isn't explicity excluded in a `.dockerignore` file.  i can only imagine this would be an issue when using containers to build containers in say ci/cd environments, but in those cases it seems unlikely the dockerfile would be the same.  so this is tied back to trying to avoid any potentially breaking changes.  i'm happy to defer to you on the reasonability of these scenarios and remove the `buildkitIgnore` parameter.